homeassistant/components/wallbox/coordinator.py

```diff
@@ -203,6 +204,43 @@ async def async_pause_charger(self, pause: bool) -> None:
         await self.hass.async_add_executor_job(self._pause_charger, pause)
         await self.async_request_refresh()
 
+    @_require_authentication
+    def _get_schedules(self) -> dict[str, Any]:
+        """Get wallbox schedules."""
+        result = self._wallbox.getChargerSchedules(self._station)
+
+        new_record: dict = {"schedules": []}
```

Comment:
please use more descriptive names (new-record and result say nothing about the content)